disaster-recovery.md

```diff
@@ -0,0 +1,169 @@
+
+**Note**: It is generally hard to detect a key compromise; this is normally discovered only through a general compromise (in which case all keys in working memory should be considered compromised), or by someone else using the key. In the specific case of the Tendermint key, a double signing could also stem from a misconfiguration or an error in switching between validator replicas; thus, a single double signing by a validator should be investigated, but does not require disaster recovery right away.
+
+1. Inform other validators that your tendermint key has been compromised and switch off your node.
+2. At network restart, add the new tendermint pubkey in the [genesis.json](https://github.com/vegaprotocol/networks#genesis-config-for-validator) and join others on a LNL [checkpoint restart](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+
+
+### Scenario 1.5: Several keys compromised in a short period of time
+
+In the event several keys are compromised/found to be compromised in a short time, the network is under a (successful) attack from a competent and dedicated attacker (APT: Advanced persistent attacker). This means we should assume the worst case scenario, that the network is under attack and the attacker is after all keys.
+
+1. Disconnect all servers you can that had any access to any of the three keys. 
+2. Start a very thorough forensic audit of *everything* to ensure you’re still safe.
+
+
+### Scenario 1.6: Validator (and their keys) permanently gone, i.e. bankrupt
+
+In the event a validator disappears, their multisig key is lost (or, worse, sold during the  bankruptcy proceedings when someone else buys the HSM. Yes, this has happened in the past). To make sure this does not escalate into the next scenario, the other validators need to remove that signer reasonably quickly (a matter of one or two days) to reset the thresholds.
+
+1. Call `remove_signer` on the [MultisigControl Contract](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e#writeContract) with the affected key
+2. Find a reliable validator to replace this party and agree this validator using appropriate validator governance processes
+3. Call `add_signer` to add a new validator on the [MultisigControl Contract](https://etherscan.io/address/0x9d0707C91C67d598808834b4881348684e92E11e#writeContract) 
+
+
+### Scenario 1.7: Loss of control of MultiSig contract on Ethereum
+
+In the event that t+1 validators lose their ETH key/go bankrupt/disappear without trace, and the multisig contract is not adapted fast enough, we lost control of the contract. As there’s limited assets on three for the time being, Vega can be reborn with a new instance.
```

Comment:
@Vegaklaus I think `t` may be undefined here.

Reply:
t= maximum number of validators we can tolerate to be corrupted (i.e., the largest integer less than a third of the total)

Reply:
👍